src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.html

```diff
@@ -194,6 +194,25 @@ <h2 *ngIf="isExamMode">
                                 <span class="d-none d-md-inline" jhiTranslate="artemisApp.assessmentDashboard.exerciseTeams">Exercise Teams</span>
                             </button>
                         </ng-template>
+
+                        <span class="col-lg-3 col-md-6 col-sm-6 align-self-center" *ngIf="isExamMode">
+                            <ng-container *jhiHasAnyAuthority="['ROLE_ADMIN', 'ROLE_INSTRUCTOR']">
+                                <ng-container *ngIf="exam && exam!.numberOfCorrectionRoundsInExam && exam.numberOfCorrectionRoundsInExam > 1">
```

Comment:
Why are the two nested ng-container needed here?

Reply:
I removed it